kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Client.java

```diff
@@ -48,5 +50,19 @@
    */
   boolean supportsApiPath(String path);
 
+  @Override
   void close();
+  
+  /**
+   * Returns the api groups
+   * @return the {@link APIGroupList} metadata
+   */
+  APIGroupList getApiGroups();
```

Comment:
Do you think we should also add another method for fetching a single APIGroup with name?:
```java
APIGroup getApiGroup(String name)
```

Reply:
@rohanKanojia I'm fine with adding that.